### PR TITLE
ZSUI-850: Display & Edit JSON config, Added configuration side-panel

### PR DESCRIPTION
**Pull request by php1990**

### CODE REVIEW

File: demo.js [hunk 1]
```diff
@@ -105,12 +105,63 @@ function cellDialog(element, event) {
...
+	config=editor.get();
+	
+	initGridUI();
+}
+
+var initGridUI = function(){
 	createGrid();
 	createStyles();
 
 	$('.cell').on('click', function(event) {
```
Comment (MikeVaz) [inline demo.js]: We don't need this anymore since slidebar replaces original configuration panel

File: demo.js [hunk 2]
```diff
@@ -105,12 +105,63 @@ function cellDialog(element, event) {
...
+
+var saveEditedJSON = function(){
+	$("#jsonContainer").fadeOut();
+
+	config=editor.get();
+	
+	initGridUI();
+}
+
+var initGridUI = function(){
```
Comment (MikeVaz) [inline demo.js]: This is now more like building a grid.

File: demo.js [hunk 3]
```diff
@@ -105,12 +105,63 @@ function cellDialog(element, event) {
...
 
 	$('.cell').on('click', function(event) {
 		cellDialog(this, event);
 	});
+}
 
+$(function() {	
+	initGridUI();
+
+	var slider = $("#menu-bar").slideReveal({
```
Comment (MikeVaz) [inline demo.js]: This is actual initialization of UI.

**Comment by MikeVaz** [comment]
Looks good. I like it.

File: demo.js (hunk 2 above)
Comment (php1990) [reply demo.js]: Maybe we should rename the function to buildGrid().

File: demo.js (hunk 1 above)
Comment (php1990) [reply demo.js]: Yeah, we can remove this snippet since it's no longer needed.

**Pull request merged**